calc.c

```diff
@@ -1,54 +0,0 @@
-#include<stdio.h>
-int main (){
-    float key;
-    printf("CALCULATOR BY SAKSHAM  \nUSER MANUAL: PRESS KEYS ACCORDINGLY \nTO ADD PRESS 1\nTO SUBTRACT PRESS 2\nTO MULTIPLY PRESS 3\nTO DIVIDE PRESS 4\nTO LEAVE PRESS 5 \nPRESS KEY (1-5) = ");
-    scanf("%f",&key);
-    for(int i = 1;)
-    if (key==1){
-        float x,y,add;
-        printf("enter two numbers x and y to add");
-        printf("\nENTER THE VALUE OF x=");
-        scanf("%f", &x);
-        printf("ENTER THE VALUE OF y=");
```

Comment:
reduce this part, same code is written 4 times

Reply:
done